CoEpi/domain/model/CENReport.swift

```diff
@@ -0,0 +1,8 @@
+struct CENReport: Codable {
+    let CENReportID: String
+    let report: String
+    let reportMimeType: String
+    let reportTimestamp: Int64
+    let CENKeys: String
+    let isUser: Bool
```

Comment:
Is this old or very recent? Report seems to have only 3 fields? https://github.com/Co-Epi/coepi-backend-go

Reply:
I guess we should develop against https://github.com/Co-Epi/data-models/pull/4/files#diff-5fb097d2df59c1b9d327693c1242ae4dR101?  But on the other hand it sounds like reports will change over time https://coepi.slack.com/archives/CUQGSQQ3A/p1585659605266100?thread_ts=1585638001.264000&cid=CUQGSQQ3A

Regardless I think we can remove the ones that aren't in the API definition